README.md

```diff
@@ -134,7 +135,8 @@ If you customised your `.babelrc`-file you might need to manually add
 `kcd-scripts` will automatically load any `.ts` and `.tsx` files, including the
 default entry point, so you don't have to worry about any rollup configuration.
 
-`tsc --noemit` will run during lint-staged to verify that files will compile.
+`tsc --build tsconfig.json` will run during before committing to verify that files will compile.
+So make sure to add the `noEmit` flag to the `tsconfig.json`'s `compilerOptions`.
```

Comment:
Is there a reason for this requirement to add noEmit to the config? Could we not use the flag in the command instead?

Reply:
Unfortunately, no. 
When you build a typescript project, you have two options:

1. Use a tsconfig.json file (that's what we do here)
2. Provide **all** compiler options as command line options

And you can't mix those two.
So running something like `tsc --noEmit --build tsconfig.json` will not work (`error TS6369: Option '--build' must be the first command line argument.`) and switching noEmit and build will also not work (`error TS5072: Unknown build option '--noEmit'.`).

I thought about creating a custom tsconfig file just before building the ts project. It could use the `extends` field to include all the configuration of the project's tsconfig. But I'm not sure if this is that reliable. For example it needs to be removed again after building the project. But what happens if the user interrupts the script during the ts build? These cases would need to be handled. Handling that would require quite some code. And I'm not sure if adding this code is really worth it. Maybe having this hint (i.e. "please add --noEmit to your tsconfig file") is enough. After all, you will want to have this option set anyways if you use babel as a typescript compiler.

Reply:
I think eventually we'll have a tsconfig in kcd-scripts that my projects will extend. But for now this is acceptable. Thanks!